Doc/library/struct.rst

```diff
@@ -367,6 +360,12 @@ Notes:
    For the ``'E'`` and ``'C'`` format characters, the packed representation uses
```

Comment:
Some PR has to update E and C in this line, but I'm not sure which one is doing it.  Just making sure you did not forget.

Reply:
Thanks, this was missed in #132827.

Reply:
https://github.com/python/cpython/pull/133249